src/lexer/select.ts

```diff
@@ -28,6 +28,9 @@ class Select implements ILexer {
       });
     });
 
+    /**
+     * Does query need to be returned?
```

Comment:
It probably doesn't need to be returned since I think it gets modified in place. Not saying that's ideal really either, it'd probably be best to pass a Query object and get a new one out.

Reply:
It's passed by reference so I personally would just return void. I know you don't like that though ;)

Reply:
I guess you can make it return `void` :'(

Reply:
Not if you're unhappy with that. It's your repo (´･_･`)

Reply:
Leave it then >:D